### ISSUE TITLE
Conditional keywords ignored if keyword doesn't exist

### ISSUE THREAD

**Issue by davidjb** [jcu-eresearch/matrix-helpers#35]
In this code,
```
%begin_asset_file_type^eq:svg%
  <img class="card-img-top img-fluid" src="%asset_url%" alt="%asset_file_type%">
%end_asset%
```

if `asset_file_type` doesn't exist in this context, the block evaluates as true, which is completely unexpected!

This has been raised on the forums, but almost certainly needs to go the Squiz Map.  See here: https://forums.squiz.net/t/conditional-keywords-ignored-if-keyword-doesnt-exist/9133

**Comment by davidjb** [comment]
~~Fixed in  5.3.5.2, 5.4.0.2, 5.5.0.0.  Should be picked up on the next upgrade.~~

**Comment by davidjb** [comment]
This is actually broken again in at least 5.4.0.3 -- found where our asset related image renderer (paint layout) was attempting to check for `%else_begin_asset_file_type^eq:svg%` and that was evaluating as `true` for a Site asset, which clearly has nothing to do with files or SVGs.  Anyway, this issue is still a bug, but we can work around it with SSJS, which is much better than the awful rendering of conditional keyword logic.

Feel free to reopen if there's a use case for conditional logic, but otherwise, let's try and bury this one.

Like all good Matrix bugs, a fix in one area broke another.
